### ISSUE TITLE
Error when requiring module

### ISSUE THREAD

**Issue by bkconrad** [node-inspector/v8-profiler#61]
when I `require('v8-profiler')`, I receive the following:

```
Error: Cannot find module '/home/kaen/code/myproject/node_modules/v8-profiler/build/profiler/v5.2.9/node-v14-linux-x64/profiler.node'
```

The mentioned file indeed does not exist, but the following *does* exist:
```
/home/kaen/code/myproject/node_modules/v8-profiler/build/profiler/v5.2.9/node-v11-linux-x64/profiler.node
```

Notice `v14` vs `v11`

System Info:
```
$ node --version
v0.12.2

$ npm list | grep profiler
v8-profiler@5.2.9

$ uname -a
Linux kaen-desktop 3.13.0-46-lowlatency #77-Ubuntu SMP PREEMPT Mon Mar 2 18:49:29 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -c
Codename:	trusty
```

Thanks for any help,
Bryan

**Comment by 3y3** [comment]
Are you using `v8-profiler` with node-inspector?
Are you tried to reinstall package?
Are you have other versions on nodejs installed? (Or any programms which embedds nodejs - `nw.js`, `electron`)

**Comment by Kevin-Allain** [comment]
Hello, I am facing the same issue, with the same infos coming from my terminal concerning the version and npm list | grep profiler.
I also tried to change the folder name to node-v0.9.9-pre-darwin-x64, which resulted in the following error:

dyld: lazy symbol binding failed: Symbol not found: _node_module_register
  Referenced from: /Users/kevin/Documents/cornerstone-2.0.6/examples/MixedJavaScriptExample/node_modules/v8-profiler/build/profiler/v5.2.9/node-v0.9.9-pre-darwin-x64/profiler.node
  Expected in: dynamic lookup
<pre><code>
dyld: Symbol not found: _node_module_register
  Referenced from: /Users/kevin/Documents/cornerstone-2.0.6/examples/MixedJavaScriptExample/node_modules/v8-profiler/build/profiler/v5.2.9/node-v0.9.9-pre-darwin-x64/profiler.node
  Expected in: dynamic lookup

Trace/BPT trap: 5
</code></pre>

I also use npm with d3, with the following line 
<pre><code> var d3 = require("d3");
</code></pre>
which works fine.

**Comment by keepitsimple** [comment]
I have the same problem with version node.js v.0.12.7

**Comment by keepitsimple** [comment]
I bumped into the same issue until I realized that I have 2 node.js: 
/usr/bin/node (v 0.10.40)
/usr/local/bin/node (v.12.7)

**Comment by 3y3** [comment]
Should be fixed in v8-profiler@5.5.0 and node-inspector@0.12.5
